db/schema.rb

```diff
@@ -130,7 +130,7 @@
   end
 
   create_table "communications", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
-    t.uuid "appointment_id", null: false
+    t.uuid "appointment_id"
```

Comment:
As discussed offline, this is because a previous migration didn't check in its update to `schema.rb`